en/assembly/quick_start_pixhawk4.md

```diff
@@ -26,33 +30,29 @@ Pixhawk 4 should be mounted on the frame using vibration-damping foam pads (incl
 
 ## GPS + Compass + Buzzer + Safety Switch + LED
 
-Attach the provided GPS with integrated compass, safety switch, buzzer and LED to the GPS port.
+Attach the provided GPS with integrated compass, safety switch, buzzer and LED to the **GPS MODULE** port.
 
-> **Note** The diagram shows a GPS with an integrated compass.  The GPS/Compass should be mounted on the frame as far away from 
-  other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other 
-  electronics will reduce interference).
+The GPS/Compass should be mounted on the frame as far away from other electronics as possible, with the direction marker towards the front of the vehicle (separating the compass from other electronics will reduce interference).
 
-![Connect compass/GPS to Pixhawk 4](../../images/pixhawk4_compass_gps.png)
+![Connect compass/GPS to Pixhawk 4](../../assets/flight_controller/pixhawk4/pixhawk4_compass_gps.jpg) 
 
-When GPS module with integrated safety switch is connected to Pixhawk 4, by default the safety is enabled. This means the autopilot will not let you arm the vehicle. To arm the vehicle, you have to press and hold the safety switch for 1 sec to disable safety. Similarly, if the vehicle is armed and something goes wrong such that you are unable to disarm the vehicle from QGC or RC, you can press the safety switch again to disarm, enabling safety.
+> **Note** The GPS module's integrated safety switch is enabled *by default* (when enabled, PX4 will not let you arm the vehicle). To disable the safety press and hold the safety switch for 1 second. You can press the safety switch again to enable safety and disarm the vehicle (this can be useful if, for whatever reason, you are unable to disarm the vehicle from your remote control or ground station).
 
 
 
 ## Power
 
-Connect the output of the *Power Management Board* (PM board) that comes with the kit to one of the **POWER** bricks of Pixhawk 4 using a 6-wire cable. The PM input **2~12S** will be connected to your LiPo battery. The connections of Power Management Board, including power supply and signal connections to the ESCs and servos, are explained in the table below. Note that the PM board does not supply power to the servos via + and - pins of **FMU PWM-OUT**. 
+Connect the output of the *Power Management Board* (PM board) that comes with the kit to one of the **POWER** bricks of *Pixhawk 4* using a 6-wire cable. The PM input **2~12S** will be connected to your LiPo battery. The connections of Power Management Board, including power supply and signal connections to the ESCs and servos, are explained in the table below. Note that the PM board does not supply power to the servos via + and - pins of **FMU PWM-OUT**.  

```

Comment:
> Connect the output of the *Power Management Board* (PM board) that comes with the kit to one of the **POWER** bricks of *Pixhawk 4* using a 6-wire cable. 

- What is a power brick
- Where do you get the cable
- Image?

> The PM input **2~12S** will be connected to your LiPo battery. 

- What is 2 - 12S? (I know, but this should define)
- What is the PM input? Is it labeled on the diagram? If not it should be

> The connections of Power Management Board, including power supply and signal connections to the ESCs and servos, are explained in the table below. Note that the PM board does not supply power to the servos via + and - pins of **FMU PWM-OUT**.  

I think this is important, but the structure is wrong.